index.html

```diff
@@ -31,16 +32,8 @@ <h2>Quiz<span class="your">Your</span>Future</h2>
     </header>
     <!-- main page -->
     <div id="user-timer"></div>
-    <div id="user-interface" class="centered"></div>
 
-    <!-- footer -->
-    <footer>
-      <ul>
-        <li><a href="#">Privacy Policy</a></li>
```

Comment:
Why you're deleting the footer?

Reply:
I will remake css on next stage as on my laptop footer covers button